model/src/misc/raids.rs

```diff
@@ -0,0 +1,46 @@
+use serde::{Deserialize, Serialize};
+
+use crate::{BulkEndpoint, Endpoint, EndpointWithId};
+
+pub type RaidId = String;
```

Comment:
`RaidId` should be of type `(Raid)EventId` and you should define `Event::id` to be the same type

Reply:
I'm not really sure what you mean by the type should be (Raid)EventId

Aswell as that the RaidId defined there refers to the wing name itself(ex. Hall of Chains,...) and not a raid event (ex Dhuum, Conjured Amalgamate, ...).

So I can understand that I would have to define the RaidId to the Raid struct.

Reply:
Oh, I made a mistake. I wanted to mark RaidEvent